OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs

```diff
@@ -113,6 +115,8 @@ public Actor[] SendParatroopers(Actor self, WPos target, bool randomize = true,
 				}
 
 				aircraftInRange[a] = true;
+				Game.Sound.PlayNotification(self.World.Map.Rules, self.Owner, "Speech",
```

Comment:
The `IsNullOrEmpty` check is gone.

Reply:
It's not needed there because of check in function

		public bool PlayNotification(Ruleset rules, Player player, string type, string notification, string variant)
		{
			if (rules == null)
				throw new ArgumentNullException("rules");

			if (type == null || notification == null)
				return false;

			return PlayPredefined(SoundType.UI, rules, player, null, type.ToLowerInvariant(), notification, variant, true, WPos.Zero, 1f, false);
		}

Reply:
Oh, ok. Thanks.